src/python/WMCore/Services/WMArchive/DataMap.py

```diff
@@ -219,10 +222,10 @@ def convertOutput(outputList):
                 del outDict[field]
 
         for field in WMARCHIVE_CONVERT_TO_LIST:
-            if field in outDict and isinstance(outDict[field], basestring):
+            if field in outDict and (isinstance(outDict[field], str) or isinstance(outDict[field], bytes)):
```

Comment:
Same comment here, you can compare against a tuple of types: isinstance(outDict[field], (str, bytes))

Reply:
Awlays correct! I just pushed a new commit!